composer.json

```diff
@@ -31,7 +31,7 @@
         "drupal/core-recommended": "^8.8",
         "drupal/devel": "^2.1",
         "drupal/easy_breadcrumb": "^1.12",
-        "drupal/environment_indicator": "^3.7",
```

Comment:
Can you please merge in your release branch into this branch and update the .lock file because I'm having issues where this module (environment_indicator) is not found when i try and run drush commands.

Reply:
Sure!